PluginControlInterfaceGeneric/CMakeLists.txt

```diff
@@ -17,9 +19,8 @@ OPTION (NSM_IFACE_PRESENT
 
 SET(PLUGIN_LIB_NAME PluginControlInterfaceGeneric)
 
-SET(INCLUDE_FOLDER "include")
-
-SET(CONF_ROOT "${CMAKE_INSTALL_PREFIX}/etc/controllerconf")
+set(CONTROLLER_CONFIG_DIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}/${LIB_INSTALL_SUFFIX}/control"
```

Comment:
Hello Mattia-san,
Thank you for comment.
First of all my intention is to align installation folder of audiomanager with configuration location, e.g. by default audiomanager is installed in /usr/local/bin, in this case configuration file will be located to /usr/local/etc by following FHS.

And another intention is to make configuration path configurable and independently changed by CMake option to keep compatibility.

Nevertheless, ${LIB_INSTALL_SUFFIX} is accidentally inserted and not required. I should remove it.